docker.js

```diff
@@ -141,7 +144,7 @@ class DockerLambda {
     await dockerLambdaReady(this.port)
   }
 
-  async request (id, eventObject) {
+  async request (_id, eventObject) {
```

Comment:
what the `_` here? because it is unused?

Reply:
linter / typescript complained about unused function parameter so i renamed it to `_id` to get it to shut up

Reply:
good to know!